### DISCUSSION TITLE
Tier Chests

### DISCUSSION THREAD

**Discussion by FelixArgyle** [DiscordDungeons/Bugs#1010] [Ideas]
So there are Tier Chest items already made but currently have 0 usages so why not give them a use?

Tier 1 Chest -
Dropped from mob with 5% chance in Lemontree Plaza | Mob chance 25%
Chest Mob - Vaidavort
Stats - Dynamob

Tier 2 Chest -
Dropped from mob with 2% chance in Aexomor | Mob chance 25%
Chest Mob - Floki
Stats - Dynamob

Tier 3 Chest -
Dropped from mob with 1% chance in Nightmare's Heaven | Mob chance 25%
Chest Mob - Thorkell
Stats - Dynamob

Tier 1 Chest contents:
LUX - between 1 and 5
Beer - 1
Lemon - between 1 - 10
Limoncello Bottle - 1
Charisma Potion - 1

Tier 2 Chest contents:
LUX - between 5 and 10
Beer - between 1 and 3
Defense Potion - 1
Vodka Bottle - 1
Candy - 1

Tier 3 Chest contents:
LUX - between 10 and 15
Beer - between 1 and 5
Defense Potion - 1
Strength Potion - 1
Laxis Ore - 1
Candy - between 1 and 5

(Drop rates for the items undecided, base it off normal chests)

Tier 1 Key -
Craft: 5 Leptocite Ingots | 10 Iron Ingots | 5 Enhancing Stones

Tier 2 Key -
Craft: 10 Leptocite Ingots | 20 Iron Ingots | 10 Enhancing Stones

Tier 3 Key -
Craft: 15 Leptocite Ingots | 30 Iron Ingots | 15 Enhancing Stones

**Comment by Sven65** [comment]
We'll need more discussion relating to the rewards themselves. Otherwise a good idea.
